Helpers/opent2t-translator-helper-nest/js/test.js

```diff
@@ -0,0 +1,43 @@
+'user strict';
+
+var q = require('q');
+var NestHelper = require('./index');
+
+var testConfig = {
+    accessToken: 'c.qptzX0zoVN4ISpGvjL9rloyR6VnycNHRZp0Iftm0seBv5EScN1iEcH6D3M0qqqEfierIMNixmXWDVB0RmpICVS3blgY3PlvlTVbVzBjkEKQTRjxCnMlqiWlKtScbwMycTrY6RsUsV6GWv7tR',
+    deviceType: 'thermostats',
+    deviceId: 'unZG6J9hRgJLuaxpiaPtiI8FK55hfM7g'
+};
+
+var helper = new NestHelper(testConfig.accessToken);
+
+// set a single field
+helper.setFieldAsync(testConfig.deviceType, testConfig.deviceId, 'target_temperature_high_c', 23)
+    .then(result1 => {
+        console.log('set single result: ' + JSON.stringify(result1));
+
+        // get a single field
+        helper.getFieldAsync(testConfig.deviceType, testConfig.deviceId, 'target_temperature_high_c')
+            .then(result2 => {
```

Comment:
Prefix this line with `return` so that promises are properly chained. Also line 29 and 34.